constraints.txt

```diff
@@ -92,7 +92,7 @@ pytest==4.1.1
 python-dateutil==2.7.5
 pytoml==0.1.19
 pytz==2018.5
-raiden-contracts==0.8.0
+raiden-contracts==0.9.0
```

Comment:
Hmm I don't know if this is the only client change that will be needed with this version upgrade.  I suspect more places may be affected.

Reply:
According to the changelog not: https://github.com/raiden-network/raiden-contracts/blob/master/CHANGELOG.md

But we'll see. First thing is a conflict with `raiden-libs`. So we have to wait for #3393 and #3400.

Reply:
Hehe .. as you saw there was more changes -- and this is just for what the tests cover.